src/base/QXmppMixInvitation.cpp

```diff
@@ -21,6 +21,16 @@ class QXmppMixInvitationPrivate : public QSharedData
     QString token;
 };
 
+///
+/// \brief The QXmppMixInvitation class is used to invite a user to a
+/// \xep{0369}: Mediated Information eXchange (MIX) channel as defined by
+/// \xep{0407}: Mediated Information eXchange (MIX): Miscellaneous Capabilities.
```

Comment:
better use the new syntax: `\xep{0407, Mediated Information eXchange (MIX): Miscellaneous Capabilities}`, this generates nicer links